Processes/How-To-Submit-Content-to-the-Docs-Portal.md

```diff
@@ -1,26 +1,17 @@
 ## How To Submit Content to EOS Documentation Portal
 
-**Scenario**: You have written content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com) in .md format. Now what do you need to do?
+Once you have written your content to add to the [EOS Documentation Portal](https://docs.eosnetwork.com), you need to submit it for review. 
```

Comment:
How about: "Once you have written the content you wish to contribute to the..." ?

Reply:
Done